integration/indexes_compat_test.go

```diff
@@ -618,202 +445,6 @@ func TestDropIndexesCompat(t *testing.T) {
 	}
 }
 
-func TestDropIndexesCommandCompat(t *testing.T) {
```

Comment:
Moved to `indexes_command_compat_test.go` according to the testing naming convention.